common/static/common/js/common_libraries.js

```diff
@@ -1 +1,19 @@
-define(['domReady!', 'jquery', 'backbone', 'underscore', 'gettext']);
+define([
```

Comment:
Given this, I could move the cloned code into an LMS-only directory if folks think that is cleaner.